src/vmm/src/devices/mod.rs

```diff
@@ -28,6 +28,9 @@ use crate::logger::IncMetric;
 // network metrics is reported per device so we need a handle to each net device's
 // metrics `net_iface_metrics` to report metrics for that device.
 pub(crate) fn report_net_event_fail(net_iface_metrics: &NetDeviceMetrics, err: DeviceError) {
+    if let DeviceError::QueueError(QueueError::InvalidAvailIdx(iai)) = err {
```

Comment:
can we rename `iai` to `err` so that we are consistent with what we do in other places? Also, because it took me 5 seconds to understand what it was :)

Reply:
and everywhere else that this appears